src/tooltip.js

```diff
@@ -98,7 +102,12 @@ nv.models.tooltip = function() {
 
         trowEnter.append("td")
             .classed("value",true)
-            .html(function(p, i) { return valueFormatter(p.value, i, p) });
+            .html(function(p, i) {
+                if (enableComplexValueFormatter) { // Mirror Chart
+                    return valueFormatter(p, i);
+                }
+                return valueFormatter(p.value, i, p);
```

Comment:
I didn't implement `enableComplexValueFormatter` or the mirrored charts and sadly none of it is documented. So I am not sure what the intention is behind this flag. Reading the code deeply might reveal it is useless or even unnecessary.

Do I want to touch `enableComplexValueFormatter` right now? To be honest, as long as charts continue working properly I would prefer to not question much this old code. If we really care we can always come back later and improve the implementation.

Regarding the `valueFormatter` function, I do see that's the current signature on the upstream nvd3 repo. I think the idea behind this is to provide developers with the flexibility to implement formatters that might require the index of the data point and the complete data point object and not only its value. I think of this as the definition of an interface.